src/QuoteBox.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import './App.css';
+
+
+class QuoteBox extends Component {
+  handleClick(e) {
```

Comment:
use `event` instead of `e`
explicit is better than implicit
e.g `e` could be an error